internal/testplanet/satellite.go

```diff
@@ -104,7 +104,7 @@ func (planet *Planet) newSatellites(count int) ([]*satellite.Peer, error) {
 					UptimeCount:       0,
 					AuditCount:        0,
 					NewNodePercentage: 0,
-					OnlineWindow:      time.Hour,
+					OnlineWindow:      0,
```

Comment:
@mobyvb I am not sure if its a good idea to disable the window entirely for all tests per default.
For that reason i created the above  "Reconfigure".

Reply:
Can we just merge and release it. Update the test later (if that is needed)